repo/packages/R/riak/0/config.json

```diff
@@ -0,0 +1,102 @@
+{
+  "type": "object",
+      "properties": {
+        "riak": {
+          "type": "object",
+          "depscription": "Riak Mesos Framework specific properties",
+          "properties": {
+            "master": {
+              "type": "string",
+              "description": "The URL of the Mesos master. The format is zk://host1:port/mesos.",
+              "default": "zk://master.mesos:2181/mesos"
+            },
+            "zk": {
+              "type": "string",
+              "description": "Zookeeper address. Specify in the format host:port.",
+              "default": "master.mesos:2181"
+            },
+            "ip": {
+              "type": "string",
+              "description": "Framework HTTP API bind address."
+            },
+            "hostname": {
+              "type": "string",
+              "description": "Framework HTTP API hostname."
+            },
+            "log": {
+              "type": "string",
+              "description": "Framework log file location."
+            },
+            "user": {
+              "type": "string",
+              "description": "Framework User.",
+              "default": "root"
+            },
+            "framework-name": {
+              "type": "string",
+              "description": "Framework Instance Name.",
+              "default": "riak"
+            },
+            "role": {
+              "type": "string",
+              "description": "Framework Role.",
+              "default": "*"
+            },
+            "url": {
+                "type": "string",
+                "description": "Riak Mesos Framework download URL",
+                "default": "http://riak-tools.s3.amazonaws.com/riak-mesos/riak_mesos_linux_amd64_0.1.0.tar.gz"
```

Comment:
please update the config file so that a separate riak.json file is not required for installation.